core/backend/src/IModelDb.ts

```diff
@@ -2772,7 +2772,7 @@ export class BriefcaseDb extends IModelDb {
           Logger.logWarning(loggerCategory, `The CodeService is not available for this briefcase: errorId: ${(e as CodeService.Error).errorId}, errorMessage; ${e.message}. Proceeding with BriefcaseDb.open(), but all operations involving codes will fail.`);
           briefcaseDb._codeService = {
             verifyCode: (props: CodeService.ElementCodeProps) => {
-              if (props.props.code !== null) {
+              if (props.props.code !== Code.createEmpty()) {
```

Comment:
```suggestion
              if (!Code.isEmpty(props.props.code))) {
```

Reply:
Resolved in [commit ff9178df8b77156c34ed76e2b113a7d39811ef28](https://github.com/iTwin/itwinjs-core/pull/6776/commits/ff9178df8b77156c34ed76e2b113a7d39811ef28)